query.go

```diff
@@ -24,11 +26,41 @@ func skippedQueryContext(_ context.Context, _ string, _ []driver.NamedValue) (dr
 	return nil, driver.ErrSkip
 }
 
+// add Custom Lables to metrics and traces
+type ctxKey string
+
+const (
+	meterCtxKey  ctxKey = "meterCtxKey"
+	tracerCtxKey ctxKey = "traceCtxKey"
+)
+
+func getLabels(ctx context.Context, key ctxKey) []attribute.KeyValue {
+	labels, ok := ctx.Value(key).([]attribute.KeyValue)
+	if !ok || labels == nil {
+		labels = make([]attribute.KeyValue, 0)
+	}
+	return labels
+}
+
+func addLabels(ctx context.Context, key ctxKey, labels ...attribute.KeyValue) context.Context {
+	new := append(getLabels(ctx, key), labels...)
+	return context.WithValue(ctx, key, new)
+}
+
+func AddMeterLabels(ctx context.Context, labels ...attribute.KeyValue) context.Context {
```

Comment:
We should not call it labels but we are passing attribute.
We have to change the method name to something like WithCustomAtrributes or WithAdditionalAttributes

Reply:
Done